src/python/pants/option/options_bootstrapper_test.py

```diff
@@ -422,3 +368,26 @@ def munge(bin_name: str) -> str:
     assert munge(os.path.join(build_root, "bin", "pants")) == "./bin/pants"
     assert munge("/foo/pants") == "pants"
     assert munge("/foo/bar/pants") == "pants"
+
+
+def test_file_spec_args() -> None:
```

Comment:
Now that the `--spec-file` handling code has moved to OptionsBootstrapper, its test correspondingly moves here.